I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script/chapter2/chapter2_5_the_incident.rpy", line 536, in script
    jump chapter2_6_the_consequence
ScriptError: could not find label 'chapter2_6_the_consequence'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script/chapter2/chapter2_5_the_incident.rpy", line 536, in script
    jump chapter2_6_the_consequence
  File "renpy/ast.py", line 1751, in execute
    rv = renpy.game.script.lookup(target)
  File "renpy/script.py", line 918, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'chapter2_6_the_consequence'.

Windows-10-10.0.19041
Ren'Py 7.4.6.1693
Eternal Bonds 0.2
Mon Dec  6 12:30:14 2021
